db/migrate/20150413194512_remove_rendered_notes.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveRenderedNotes < ActiveRecord::Migration
+  def self.up
+    remove_column :todos, 'rendered_notes'
+  end
+
+  def self.down
+    add_column :todos, 'rendered_notes', :text
+    
+    # Call save on each todo to force generation of rendered_notes
+    # Copied from 20120412072508_add_rendered_notes
+    say "Rendering todo notes..."
+    Todo.all.each{ |todo| todo.save(validate: false) }
```

Comment:
I'd rather consider this an irreversible migration than depend on the model in a migration.